src/ocamlorg_frontend/layouts/package_layout.eml

```diff
@@ -22,7 +22,7 @@ Layout.render
       <div class="flex justify-between flex-col md:flex-row border-b border-gray-200">
         <div class="flex flex-col items-baseline mb-6">
           <h4 class="font-bold md:whitespace-nowrap flex items-center">
-            <a class="flex items-center" href="<%s Url.package_with_version package.name package.version %>">
+            <a class="flex items-center" href="<%s Url.package package.name %>">
```

Comment:
What happens when we are on a versioned URL of the package and want to go back to the versioned overview page? I think that changing the link to `Url.package package.name` is not the behaviour the user would expect here.